test_util/aws.py

```diff
@@ -500,15 +500,15 @@ def get_host_ips(self):
                   'us-east-1': 'ami-fa9b9390',
                   'us-west-1': 'ami-12b3ce72',
                   'us-west-2': 'ami-edf11b8d'},
```

Comment:
We don't. The 7.3 AMI with prereqs is based on a 7.2 base AMI where we run `yum -y --releasever=7.3.1611 update`